.github/workflows/c-embedded.yml

```diff
@@ -0,0 +1,28 @@
+name: C embedded
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+env:
+  # 2020.11
+  vcpkgGitRef: 0bf3923f9fab4001c00f0f429682a0853b5749e0
```

Comment:
What does this mean?

Reply:
Like the other stuff: Copied from `all-misc.yml` and `all-targets.yml`.

Reply:
Used by Windows runners only.